### PR TITLE
Add description to rake tasks [Fixes #116972425]

### PR DESCRIPTION
**Pull request by Maroo-b**
- Added missing description to rake tasks (https://www.pivotaltracker.com/story/show/116972425)
- Added simple tasks  log output to console.
@tansaku I saw that the Feature records are created via different migration files which cause this issue (https://github.com/AgileVentures/LocalSupport/issues/294). 
So what do you think to move the Feature records creation to db/seed.rb file ?

### CODE REVIEW

**Comment by tansaku** [comment]
@Maroo-b I'm not sure that #294 is caused by the feature records creation being in the migrations per se.  We shouldn't be changing migrations retro-actively if possible.

I would have thought the better solution was to ensure that our setup step was correctly overriding the default setup ...

File: lib/tasks/emails.rake [hunk 1]
```diff
@@ -5,13 +5,15 @@ begin
     namespace :import do
       desc 'Import email addresses from a named CSV file'
       task :emails, [:file] => :environment do |t, args|
+        puts 'Start emails import'
```
Comment (tansaku) [inline lib/tasks/emails.rake]: should we prefer rails log here?

**Comment by Maroo-b** [comment]
@tansaku I updated the PR to use Logger instead of `puts`.  
For the migrations I totally agree that we shouldn't change the structure of existing ones. But in our case we are creating records from the migration can you check those files:
- https://github.com/AgileVentures/LocalSupport/blob/develop/db/migrate/20140507165854_create_features.rb#L8
- https://github.com/AgileVentures/LocalSupport/blob/develop/db/migrate/20160129164234_add_doit_volunteer_opportunities.rb#L3
- https://github.com/AgileVentures/LocalSupport/blob/develop/db/migrate/20150108200046_create_proposed_organisation_edits.rb#L17
- https://github.com/AgileVentures/LocalSupport/blob/develop/db/migrate/20151209125200_add_search_input_bar_on_org_pages.rb#L3

So as a solution we have to improve the db/seed.rb by creating the Feature records if not existing to fix the setup step ( and avoid running migration in production env)

**Comment by tansaku** [comment]
creating the records in the migrations is intentional.  It's to ensure that those features get created in production where we can't run setup.

Moving those creations to db:seed means we can't auto-create those features in production.  I think the solution to #294 should be to have the correct seed operation run - anyhow - let's take the discussion to that ticket

**Pull request merged**